src/texstudio.cpp

```diff
@@ -1206,7 +1206,7 @@ void Texstudio::setupMenus()
 	newManagedAction(submenu, "remHLine", tr("Remove \\hline", "table"), SLOT(remHLineCB()));
 	newManagedAction(submenu, "insertTableTemplate", tr("Remodel Table Using Template", "table"), SLOT(insertTableTemplate()));
 	newManagedAction(submenu, "alignColumns", tr("Align Columns"), SLOT(alignTableCols()), QKeySequence(), "alignCols");
-	submenu = newManagedMenu(menu, "magicComments", tr("Add magic comments ..."));
+	submenu = newManagedMenu(menu, "magicComments", tr("Add magic comments"));
```

Comment:
there is no real benefit for this change, but all translations are buggered. So, I prefer to not change it.